auth0/src/main/java/com/auth0/android/provider/CustomTabsController.java

```diff
@@ -124,6 +124,7 @@ public void run() {
                 Log.d(TAG, "Launching URI. Custom Tabs available: " + available);
 
                 final Intent intent = new CustomTabsIntent.Builder(session.get())
+                        .setShowTitle(true)
```

Comment:
Ok. I'll add in this same PR a builder/style to customize this and the toolbar color.